sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -93,7 +93,7 @@ export class BatchingReceiver extends MessageReceiver {
       );
     }
 
-    await this._batchingReceiverLite.close(connectionError);
+    this._batchingReceiverLite.close(connectionError);
```

Comment:
Looking at this now it seems 'close' is a bit misnamed - it's more like "cancel" or something of that nature. (ie, it doesn't change the link state at all).

Reply:
(not something you need to fix in this PR, just curious if you agree)

Reply:
Hmmm... maybe `terminate`? (since it refers to ending the things than canceling or closing)

Reply:
Logged https://github.com/Azure/azure-sdk-for-js/issues/13390